crates/re_viewport/src/viewport_blueprint_ui.rs

```diff
@@ -459,13 +483,19 @@ fn blueprint_row_with_buttons(
 
     // The main button might have been highlighted because what it was referring
     // to was hovered somewhere else, and then we also want it highlighted here.
-    if button_hovered || main_button_response.highlighted() {
+    if button_hovered || main_button_response.highlighted() || selected {
```

Comment:
is checking for `main_button_response.highlighted()` even necessary now that `selected` is passed in? I'd expect that any time `main_button_response.highlighted()` is true, `selected` is true as well.

Reply:
Things can be highlighted without being selected by virtue of the corresponding entities being hovered over in the timeline/stream tree, so this check is indeed necessary.